t/42-screenshots.t

```diff
@@ -85,4 +87,13 @@ subtest 'screenshots are unique' => sub {
     is $whatever[1], undef, 'no second result';
 };
 
+subtest 'no errors in database log' => sub {
+    my $prep = $schema->storage->dbh->prepare('select pg_current_logfile()');
+    $prep->execute;
+    my $db_log_file  = File::Spec->catfile($ENV{TEST_PG} =~ s/^.*host=//r, $prep->fetchrow_arrayref->[0]);
```

Comment:
It's a little odd you use `File::Spec` on this line and `Mojo::File` on the next. 😉

Reply:
https://github.com/os-autoinst/openQA/pull/2600